code/__DEFINES/stat.dm

```diff
@@ -33,3 +33,8 @@
 #define SHUTTLE_STRANDED 4
 #define SHUTTLE_ESCAPE 5
 #define SHUTTLE_ENDGAME 6
+
+//Movement modes
+#define WALK 0

```

Comment:
MrPerson wanted these to match the names of the sprites
eg: 
#define WALK "walking"
#define RUN "running"
to avoid the trinary operator, plus we can do things like "\he is [M_intent]" for "he is walking" etc.